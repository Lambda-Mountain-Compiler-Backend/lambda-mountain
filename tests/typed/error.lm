
f := Î»(: a Atom). a;

f ();
