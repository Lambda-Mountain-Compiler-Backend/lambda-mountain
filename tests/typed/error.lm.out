(TypeError ((InvalidArgument (-> (Atom Atom))) Nil))
